docs/user/guides/content/index.rst

```diff
@@ -0,0 +1,82 @@
+How-to guides: content, themes and SEO
```

Comment:
This is a weird title, and seems unrelated?

Reply:
You mean than "content, themes and SEO" is a bad category?

Reply:
I'd rather represent each of these nouns than the alternatives:

1. Represent important nouns that the reader will recognize.
2. Hide everything in an umbrella term like "content" or "writing documentation", I don't really like that.. it's ambiguous.
3. Split up into separate sub-levels. Not to add more sub-levels at this stage IMO.

But if there's better wording for this, that'd be great!

Reply:
I just don't understand how these 3 things are related, and why only these things? It's a different style than all the others, so we should make it similar.

Perhaps:

```suggestion
How-to guides: Improving user experience
```

Or something?

Reply:
I've noted to get back to this!